constellations/orion/src/router/handler.go

```diff
@@ -109,6 +109,15 @@ func (h *Handler) SetupApiEndpoints() {
 		apiAFH.DELETE("/afh/:id", controllers.DeleteAFH)
 	}
 
+	apiTransaction := h.Engine.Group("api/transactions")
+	{
+		apiTransaction.GET("/all", controllers.GetAllTransactions)
```

Comment:
Let's add another endpoint (GET) that grabs ALL payment types. It can be at: `api/transactions/types`